api/src/pcapi/scripts/educational/update_educational_institutions_deposits.py

```diff
@@ -24,12 +24,12 @@ def update_educational_institutions_deposits(filename: str, path: str = DEFAULT_
         if not headers or "UAICode" not in headers or "depositAmount" not in headers:
             print("\033[91mERROR: UAICode or depositAmount missing in CSV headers\033[0m")
             return
-        _process_educational_csv(csv_rows)
+        _process_educational_csv(csv_rows, year)
     return
 
 
-def _process_educational_csv(educational_institutions_rows: Iterable[dict]) -> None:
-    currentYear = datetime.now().year
+def _process_educational_csv(educational_institutions_rows: Iterable[dict], year: int = None) -> None:
+    currentYear = year if year is not None else datetime.now().year
     try:
         educational_year = educational_repository.get_educational_year_beginning_at_given_year(currentYear)
     except exceptions.EducationalYearNotFound:
```

Comment:
Peut-être qu'en dessous dans les logs on peut du coup être plus explicite, du genre:
` print(
            f"Please add educational years in database as no educational year has been found beginning at year {current year}\033[0m"
        )`

Reply:
wdyt?